### ISSUE TITLE
As a CIF Ops team member, I want to enter previous project revisions at data intake time, so that I have access to historical project data from the start

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-cif#677]
#### Description:

This is about entering past revisions / amendments at intake time
We need to discuss how we're going to support that

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by nanyangpro** [comment]
**3 Amigos meeting with Jesse and Amanda:**
- Amendments change FA
- No full history in record (not necessary as well, implementation efforts is a concern too at this stage)
- All versions of documents are expected to be uploaded to the app tho
- Need comment box (for summary of revision history)
- Helpful to track why and when decisions were made
- The discussion only covers the previous proposals/projects
- For the 2019 data entry (soft launch), will enter with current status with a summary of revision history in general comment box

**Comment by nanyangpro** [comment]
You are right @LindsayMacfarlane, the implementation of it is required before the soft launch. A general comment box would satisfy the need as previously discussed, happy to work with the devs on determine how to title it. and where to put it.

**Comment by nanyangpro** [comment]
FYI @LindsayMacfarlane - As discussed with the devs, no wireframe is needed for this ticket, thus removed the "UX Design" and "Need Wireframe" labels.
It will be implemented before the soft launch.

**Comment by matthieu-foucault** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/677?planning-poker) with ZenHub @gurjmatharu
